### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by latifa-wakili**
My  HTML contains an input element with an id of "number" a button with an id of "convert-btn"  and a paragraph with an id of "output".

on  CSS styles I moved the page to center the content and add basic styles

The JavaScript includes:
   - An event listener for the button click to handle the conversion process.
   - Input validation to ensure the number is within the valid range (1 to 3999).
   - A function convertToRoman that converts a number to its Roman numeral equivalent.

### CODE REVIEW

**Comment by latifa-wakili** [comment]
@Sumita-BLH

**Comment by S-BLH** [comment]
you implemented all the necessary things for the project, well done latifa jan.

**Comment by latifa-wakili** [comment]
@ahzamir

File: script.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      { value: 40, numeral: 'XL' },
+      { value: 10, numeral: 'X' },
+      { value: 9, numeral: 'IX' },
+      { value: 5, numeral: 'V' },
+      { value: 4, numeral: 'IV' },
+      { value: 1, numeral: 'I' },
+    ];
+
+    let result = '';
+    for (let i = 0; i < romanNumerals.length; i) {
```
Comment (ahzamir) [inline script.js]: - Your loop runs unlimited time as you did not change the array order in your loop try this line instead:
  - Copy this line and paste it to your code to solve the issue.
```javascript
for (let i = 0; i < romanNumerals.length; i++) {
```

- In here I added `i++` to change the array loop order while it runs to + 1

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+The JavaScript includes:
+
+An event listener for the button click to handle the conversion process.
+Input validation to ensure the number is within the valid range (1 to 3999).
+A function convertToRoman that converts a number to its Roman numeral equivalent.
+
+
+## Demo 📸
+![Demo]
+(Roman-NC.png)
```
Comment (ahzamir) [inline README.md]: - Add your project Demo Image here using this line of code:

```
![Demo](HERE YOUR PROJECT IMAGE LINK)
```

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+## Technologies Used 🛠️
+
+- HTML
+- CSS
+-JS
```
Comment (ahzamir) [inline README.md]: Add a space between `-JS` make it `- JS`

File: README.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+```bash
+# git@github.com:latifa-wakili/Roman-Numeral-Converter.git.
+```
+
+```bash
+# cd Roman-Numeral-Converter.
+```
+
+# vscode.
+```
```
Comment (ahzamir) [inline README.md]: - Remove this and paste this instead:
  - Dont forget to remove the parenthesis after you pasted `( )`

```
(```
code .
```)
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
@latifa-wakili 

- Although you have done a great job by implementing lots of requirements but still your project has some issues check the below comments for more details:

**Comment by latifa-wakili** [comment]
@ahzamir bcz of  JS linters erros i changed some of my codes pls rereveiw it now

**Comment by ahzamir** [comment]
@latifa-wakili 

- Your code is not functioning well and it is not working properly.

**Comment by latifa-wakili** [comment]
@ahzamir finally I solved my linters error and my code works 
please rereview my repo for approval

**Review by ahzamir — APPROVED** [review]
### Hi @latifa-wakili 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Don't forget to deploy your project and share on Social Media.
- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Roman Numeral Converter project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️Your repo have been reviewed by your partner
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**